### ISSUE TITLE
Ordering of sections

### ISSUE THREAD

**Issue by jpfeuffer** [OpenMS/OpenMS-docs#59]
We should probably think about the order of the doc sections. I just looked at the headings in the Sidebar and I felt like the order is not always optimal. E.g. TOPP tools are explained after the TOPP Tutorial. And Scripting with TOPP might be better suited in the TOPP section.

## Read Existing Documentation

I have checked the following places for the error:

- [ ] [OpenMS Documentation Troubleshooting]()
- [ ] [OpenMS API Developer FAQs]()

## Create OpenMS Bug Report

I have created the bug report in OpenMS for any pipeline errors.

- [ ] [OpenMS Open Issues](https://github.com/OpenMS/OpenMS/issues)

## Description of the documentation bug

- [ ] I have attached screenshots of the incorrect documentation

## OpenMS Installation

- Version:

**Comment by tapaswenipathak** [comment]
> TOPP tools are explained after the TOPP Tutorial

In terms of content for readers [TOPP Tutorial](https://staging-openms.readthedocs.io/en/staging/docs/tutorials/TOPP/TOPP-tutorial.html) has more advanced and informative content than [TOPP Tools](https://staging-openms.readthedocs.io/en/staging/docs/topp/topp.html).

<img width="240" alt="Screenshot 2022-05-29 at 11 57 30 PM" src="https://user-images.githubusercontent.com/4143778/170885916-35766e04-0f93-454d-a795-59f24d244ac9.png">

OpenMS TOPP tools just give an introduction and a redirect to API reference. I would say -1.

> And Scripting with TOPP might be better suited in the TOPP section.

The focus for users is TOPP Tutorial and not TOPP section.  Do it in one go or together as we have parallel equal weightage focus point for readers in [API reference](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/nightly/html/TOPP_documentation.html). -1.

**Comment by ghost** [comment]
I agree with @jpfeuffer. I think that the tutorials should go after the tools description section.
This is an example:
<img width="287" alt="Screen Shot 2022-05-30 at 4 11 45 pm" src="https://user-images.githubusercontent.com/31978630/170932221-416b2ac3-c7ac-49fa-897d-e8df72e8c92d.png">

**Comment by ghost** [comment]
ntroduction

What is OpenMS
Background
Getting Started

Installation on GNU/Linux
Installation on macOS
Installation on Windows
Quick Start Guides

User Quick Start Guide
Contributor Quick Start Guide
TOPP

What is TOPP
Key Concepts
TOP tools
Graphical tools
File handling tools
Signal processing and preprocessing tools
Quanitation tools
Map Alignment tools
Protein/Peptide identification tools
etc
KNIME

What is KNIME
User interface
Examples
TOPPAS

What is TOPPAS
User Interface
Examples
Tutorials

TOPP
KNIME
TOPPAS
Developer Resources

Developer guidelines for adding new dependency libraries
Link external code to OpenMS
Custom compilation of OpenMS
Build custom KNIME application
Add new tool to the TOP suite
Additional Resources

OpenMS git workflow
Reporting bugs and issues
Write and label github issues
Pull request checklist
Downloads

OpenMS installers
Workflows
OpenMS releases
Other resources
Quick Reference

Contributor FAQ
Developer FAQ
Glossary
Contact Us

**Comment by timosachsenberg** [comment]
@jpfeuffer @greengypsy do you think it would make sense to put this into a "workflow system" section?
e.g., we will probably expand this with nextflow, snakemake and galaxy in the future. Having a top level menu entry for all of them seems a bit overkill

What is KNIME
User interface
Examples
TOPPAS

What is TOPPAS
User Interface
Examples
Tutorials

**Comment by tapaswenipathak** [comment]
# Introduction​​

- [ ] ## About OpenMS
- [ ] ## Background
- [ ] ## How OpenMS Works

# Getting Started 

- [ ] ## Installation on GNU/Linux 
- [ ] ## Installation on macOS 
- [ ] ## Installation on Windows

# OpenMS Applications and Tools 

- [ ] ## OpenMS Application 
- [ ] ## TOPP Tools 
- [ ] ### Introduction to TOPP 
- [ ] ### Key Concepts 
- [ ] ### Types of TOPP tools 
- [ ] - Graphical tools 
- [ ] - File handling tools 
- [ ] - Signal processing and preprocessing tools 
- [ ] - Quantitation tools 
- [ ] - Map Alignment tools 
- [ ] - Protein/Peptide identification tools

- [ ] ## UTILS Tools 
- [ ] ### Introduction to UTILS
- [ ] ### Key Concepts
- [ ] ### Types of TOPP tools 
- [ ] ## Command Line Interface 
- [ ] ## Visualize with OpenMS

# Run workflows with OpenMS Tools 
## OpenMS in KNIME 
(introduction): not a heading 

- [ ] ### Installation 
- [ ] ### Getting started 
- [ ] ### Workflows (link to downloads) 

## OpenMS in nextflow 
(introduction): not a heading 

- [ ] ### Installation 
- [ ] ### Getting started 
- [ ] ### Workflows (link to downloads) 

## OpenMS on Galaxy
(introduction): not a heading 

- [ ] ### Installation 
- [ ] ### Getting started 
- [ ] ### Workflows (link to downloads) 

# Tutorials and Quick Start Guides
## Tutorials 

- [ ] ### TOPPView Tut 
- [ ] ### TOPP Shell tut 

- [ ] ### User tut

## Quick Stat guides

- [ ] ### KNIME qsg
- [ ] ### Nextflow qsg
- [ ] ### Galaxy qsg 
- [ ] ### Developer
- [ ] ### Contributor
- [ ] ### User

# Develop with OpenMS 

- [ ] ## C++ 
- [ ] ## Python 
- [ ] ## OpenMS Core Library 
- [ ] ## Build OpenMS from source 
- [ ] ## Link external code to OpenMS 
- [ ] ## Developer tutorial (external/doxygen) 
- [ ] ## API reference (external/doxygen)

# Contribute to OpenMS 

- [ ] ## Build from Source 
- [ ] ## OpenMS Git Workflow
- [ ] ## Add new tool to the TOPP suite 
- [ ] ## Pull request checklist 
- [ ] ## Reporting bugs and issues (and how to label them) 

## Advanced 

- [ ] ### Developer guidelines for adding new dependency libraries 
- [ ] ### Build custom KNIME plugin

# Downloads 

- [ ] ## OpenMS installers 
- [ ] ## Workflows 
- [ ] ## OpenMS releases 
- [ ] ## Other resources

- [ ] ### Schema

- [ ] ### Data
- [ ] ### Tutorials/Guides
- [ ] ### Scripts

# Quick Reference

- [ ] ## Contributor FAQ
- [ ] ## Developer FAQ
- [ ] ## Contact Us
- [ ] ## Glossary

cc @greengypsy. 

(the doc has limited visibility restrictions)

Deps
- https://github.com/OpenMS/Tutorials/issues/205.

**Comment by tapaswenipathak** [comment]
cc @axelwalter @enetz @jpfeuffer @KyowonJeong @poshul @OpenMS/doc-team.

Please feel free to add comments/suggestions/thoughts/edits on finalized structure for the first iteration.

while i and @christinakumar prep for merge.

**Comment by enetz** [comment]
I feel like we can leave out the Utils section? In practice the Utils are just TOPP tools, same interface, same user experience. They are just usually in beta and less well tested. A sub-page under TOPP Tools describing that should be enough.
